openam-authentication/openam-auth-oath/src/main/java/org/forgerock/openam/authentication/modules/oath/OATH.java

```diff
@@ -583,6 +584,12 @@ private boolean checkOTP(String otp) throws AuthLoginException {
                 //get Time Step
                 long localTime = time;
                 localTime /= totpTimeStep;
+                long lastLoginTimeStep = lastLoginTime / totpTimeStep;
```

Comment:
This line is not necessary as it is an artifact of OPENAM-1462, which isn't present on 11.0.x.
You should just compare the lastLoginTime to the localTime

Reply:
Sorted